update readme with reverse proxy (#13)
stardog-union/launchpad-docs: README.md

> **Warning**:
> The private key to your client certificate **must** be unencrypted.

## Accessing Launchpad with HTTPS

> To enable communication between the front end (i.e., your browser) and Stardog via HTTPS, you can configure a reverse proxy, such as [nginx](https://nginx.org/en/). Once you have set up your proxy, set the `BASE_URL` variable in your `.env` file to its address. You will need to recreate your Docker containers after updating your `.env` file.

Reviewer: s/and Stardog via HTTPS/and Launchpad via HTTPS/

Also, would be good to provide a simple example config, if possible.

> **Warning**:
> The private key to your client certificate **must** be unencrypted.

## Accessing Launchpad with HTTPS

To enable HTTPS communications for Launchpad, you can configure a reverse proxy or load balancer, such as [nginx](https://nginx.org/en/). Once you have set up your proxy/load balancer, set the `BASE_URL` environment variable to its address. 

> To secure communications between Stardog and Launchpad, see [SSL Cert Verification](./README.md#ssl-cert-verification).

Reviewer 2: ```suggestion
To secure communications between Stardog and Launchpad, see [SSL Cert Verification](#ssl-cert-verification).
```

This will make it such that Github won't reload the page when clicking on the hyperlink. It'll just move up/down the document already loaded.

> **Warning**:
> The private key to your client certificate **must** be unencrypted.

## Accessing Launchpad with HTTPS

> To enable communication between the front end (i.e., your browser) and Launchpad via HTTPS, you can configure a reverse proxy, such as [nginx](https://nginx.org/en/). Once you have set up your proxy, set the `BASE_URL` variable in your `.env` file to its address. You will need to recreate your Docker containers after updating your `.env` file.

Reviewer 2: ```suggestion
To enable HTTPS communications for Launchpad, you can configure a reverse proxy or load balancer, such as [nginx](https://nginx.org/en/). Once you have set up your proxy/load balancer, set the `BASE_URL` environment variable to its address. 
```

Any time you need to change the config for Launchpad, a restart is required. This is not specific to this setting. 
Maybe mention here a link to securing communications between Stardog and Launchpad because that is not what is being described in this section but I could see someone looking for that info next or wondering "okay now the Launchpad application can be visited via HTTPS but how can I make sure that the traffic between Launchpad and my Stardog server is also encrypted via HTTPS ...".

An example nginx config file is below:
```yaml
version: '3'
services:
     nginx:
         user: root
         container_name: nginx
         hostname: nginx
         image: nginx
         restart: unless-stopped
         ports:
             - 9443:443
         volumes:
             - /path/to/conf/nginx:/etc/nginx/conf.d
             - /path/to/SSL:/etc/ssl/private
             - /path/to/nginx:/var/log/nginx
```

> 

Reviewer: Okay, this is my fault, as I asked for a sample config. Maybe just omit the sample nginx config?

> 

Reviewer: Apologies to @StevePlace68 for all the churn on this.

> 

Author: No worries
